### PR TITLE
Add symfony/process package

### PR DESCRIPTION
**Pull request by DigiLive**
Instead of calling functions exec and shell_exec, make use of the
symfony/process package.
The package will take care of the differences between operating system
and escaping arguments to prevent security issues.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0"
+        "php": "^7.3 || ^8.0",
+        "symfony/process": "^5"
```
Comment (JBlond) [inline composer.json:26]: Why not version six?

**Review by JBlond — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (DigiLive) [reply composer.json:26]: Because version 6 requires php version 8 if I'm not mistaken.

Comment (JBlond) [reply composer.json:26]: "php": ">=7.2.5",

Comment (JBlond) [reply composer.json:26]: You are right

Comment (JBlond) [reply composer.json:26]: You are right

**Pull request merged**